packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -180,8 +194,85 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         electionData.addressVoted[msg.sender] = true;
     }
 
-    function getNomineeVotes(address nominee) external view override returns (uint) {
-        return _electionStore().electionData.nomineeVotes[nominee];
+    function isElectionEvaluated() external view virtual override returns (bool) {
+        return _electionStore().electionData.isElectionEvaluated;
+    }
+
+    function evaluateElectionBatch() external virtual override {
```

Comment:
Are we checking that voting finished before this computation starts? I fear that if votes continue to move around while this computation is ongoing, it may break.

Reply:
isEpochFinished() is the end of voting phase

Reply:
Ah cool 👍